ios/Capacitor/Capacitor/JSTypes.swift

```diff
@@ -148,10 +149,7 @@ extension JSValueContainer {
     }
 
     public func getDate(_ key: String) -> Date? {
-        if let isoString = jsObjectRepresentation[key] as? String {
-            return Self.jsDateFormatter.date(from: isoString)
-        }
-        return nil
+        return jsObjectRepresentation[key] as? Date        
```

Comment:
Sorry if it wasn't clear. I meant that the multiple lines with the date checking could be coalesced into a single line. I think we need to keep the string interpretation intact so we can work with JSON strings or date objects.
```suggestion
        if let isoString = jsObjectRepresentation[key] as? String {
            return Self.jsDateFormatter.date(from: isoString)
        }
        return jsObjectRepresentation[key] as? Date        
```

Reply:
Ah, its ok, makes sense now